app.js

```diff
@@ -10,7 +10,7 @@ function Project (proj){
 }
 
 Project.prototype.toHtml = function() {
```

Comment:
**TODO FOR LOST POINTS**
This is a top level comment that you need to separate your files into separate folders, i.e. js files in ``scripts``, css files in ``styles``.